lib/order_query/point.rb

```diff
@@ -32,19 +32,19 @@ def position
     end
 
     # @return [ActiveRecord::Relation]
-    def after
-      side :after
+    def after(strict = true)
+      side :after, strict
     end
 
     # @return [ActiveRecord::Relation]
-    def before
-      side :before
+    def before(strict = true)
```

Comment:
good for me, but it's your decision. We can always implement it as an old-school hash.